seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -2703,12 +2680,12 @@
       "title": "Directed and Undirected Graphs",
```

Comment:
This may be worthy of it's own issue, but I write my thoughts here for now.

This challenge feels like a introduction challenge you're only supposed to read. Perhaps we should use the same format as the introduction to each major section? What's stopping us from using that type of challenge in the middle of a section?

The seed code is neither used nor explained and doesn't add any value to the challenge. Furthermore, the title "Directed and Undirected Graphs" suggests that this challenge covers the difference between the two types of graphs, which it doesn't.

I suggest we:
1. Update this challenge to use the same format as other introduction challenges.
2. Rename it to "Introduction to Graphs".

@freeCodeCamp/moderators What do you think?